src/js/other-websites/inpage_menu.js

```diff
@@ -0,0 +1,249 @@
+/* global restrictOrRestorePageTabbing */
+
+function iframeCloseRelayInPageMenu() {
+  // TODO: SEND MESSAGE TO CLOSE IFRAME
+  // console.log("iframeCloseRelayInPageMenu");
+}
+
+function preventDefaultBehavior(clickEvt) {
+  clickEvt.stopPropagation();
+  clickEvt.stopImmediatePropagation();
+  clickEvt.preventDefault();
+  return;
+}
+
+function getRelayMenuEl() {
+  return document.querySelector(".fx-relay-menu");
+}
+
+// let activeElemIndex = -1;
+function handleKeydownEvents(e) {
+  const relayInPageMenu = getRelayMenuEl();
+  const clickableElsInMenu = relayInPageMenu.querySelectorAll("button, a");
+
+  if (clickableElsInMenu[activeElemIndex] !== undefined && watchedKeyClicked) {
+    return clickableElsInMenu[activeElemIndex].focus();
+  }
+}
+
+function createElementWithClassList(elemType, elemClass) {
+  const newElem = document.createElement(elemType);
+  newElem.classList.add(elemClass);
+  return newElem;
+}
```

Comment:
Revised in 4344dc8e29ad93376f4af2c58eebb7a81182ed08

FWIW, this function is used on other files: https://github.com/mozilla/fx-private-relay-add-on/blob/spike-iframe-concept/src/js/other-websites/add_input_icon.js#L32-L36

It may be worth moving into utils.